### PR TITLE
UI/UX adjustments

### PR DESCRIPTION
**Pull request by clara-rthrdt**
Streamlining intro, moving design principles to "contribute" section

### CODE REVIEW

File: docs/docs/02_data-access/_category_.json
Comment (cathleenberger) [inline docs/docs/02_data-access/_category_.json:1]: It feels like this needs to be a careful balance between brief/accessible and clear/specific. The Hub doesn't currently cover data access across all fields or possibilities, but is focused on platforms (for now). I would therefore suggest to keep the headline as is.

File: docs/docs/05_literature/_category_.json
Comment (cathleenberger) [inline docs/docs/05_literature/_category_.json:1]: See above, specificity helps navigate and get started. This chapter (for now) invites links and/or lists of existing research projects -- it is illustrative. It does not provide research on/through the Hub as of yet.

**Review by cathleenberger — COMMENTED** [review]
Thank you for submitting. The move of the design principles makes sense to me and happy to merge that as suggested. I would, however, recommend to keep the headlines as is, given that the specifity provides more context.

**Comment by clara-rthrdt** [comment]
Thank you for your feedback! I agree with you and will open a new PR without changes in the headings. :)

**Pull request closed without merging**